assets/js/base/components/pagination/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Label from '../label';
+import { getIndexes } from './utils.js';
+import './style.scss';
+
+const Pagination = ( { currentPage, pagesToDisplay, onPageChange, totalPages } ) => {
+	const { minIndex, maxIndex } = getIndexes( pagesToDisplay, currentPage, totalPages );
+	const pages = [];
+	for ( let i = minIndex; i <= maxIndex; i++ ) {
+		pages.push( i );
+	}
+	const showPreviousArrow = minIndex !== 1;
+	const showNextArrow = maxIndex !== totalPages;
+
+	return (
+		<div className="wc-block-pagination">
+			<Label
+				screenReaderLabel={ __( 'Navigate to another page', 'woo-gutenberg-products-block' ) }
+			/>
+			{ showPreviousArrow && (
+				<button
+					className="wc-block-pagination-page"
+					onClick={ () => onPageChange( 1 ) }
+					title={ __( 'First page', 'woo-gutenberg-products-block' ) }
+				>
+					<Label
+						label="←"
```

Comment:
We could use those < > symbols that other blocks use if we wanted right?

Reply:
Done: 52f4e5581757b5ad13573918f0b59c4089e7f65a.